src/ecrc-api/src/main/java/ca/bc/gov/open/ecrc/controller/OauthController.java

```diff
@@ -52,12 +52,12 @@
 
 	@ResponseStatus(code = HttpStatus.FOUND)
 	@RequestMapping(value = "/protected/getBCSCUrl", method = RequestMethod.GET)
-	public ResponseEntity<String> getBCSCUrl(@RequestParam(name = "redirectUrl", required = true) String redirectUrl) throws OauthServiceException {
+	public ResponseEntity<String> getBCSCUrl() throws OauthServiceException {
```

Comment:
Do we still need this endpoint of 'getBCSCUrl' beinfg that our '/authorize' will generate a 302 and cause a redirect?

Reply:
Re-added authorized endpoint